src/common/CMakeLists.txt

```diff
@@ -19,12 +19,13 @@ target_sources(common
     "${CMAKE_CURRENT_LIST_DIR}/PartitionInfo.cc"
     "${CMAKE_CURRENT_LIST_DIR}/VerificationUtils.cc"
     "${CMAKE_CURRENT_LIST_DIR}/ReportUtils.h"
+    "${CMAKE_CURRENT_LIST_DIR}/OpensmtAssert.h"
 )
 
 install(FILES
         Integer.h Number.h FastRational.h XAlloc.h Alloc.h StringMap.h Timer.h osmtinttypes.h
-        TreeOps.h Real.h FlaPartitionMap.h PartitionInfo.h OsmtApiException.h TypeUtils.h 
-        NumberUtils.h NatSet.h
+        TreeOps.h Real.h FlaPartitionMap.h PartitionInfo.h OsmtApiException.h TypeUtils.h
+        NumberUtils.h NatSet.h OpensmtAssert.h
```

Comment:
I think you need to include `OpenSMTAssertException.h` and `OpenSMTInternalException.h` otherwise OpenSMT might not compile when included as a library in other projects.

Reply:
SMTS's client just catches the base exception and reports the message to the SMTS server

Reply:
Yes, I also think like @MasoudAsadzade that in this particular use case we're aiming at, there's no need to install the header.